### PR TITLE
Adding options for standalone reading, several use cases

### PR DESCRIPTION
**Pull request by bitoiu**
Hey @jbjonesjr,

A few improvements to the doc:
- Adding the flags on the README so a user can see what the command does without a command line
- Separating the readme in use cases (kind of pick what you need)

**WIP, so please don't comment for now.**

### CODE REVIEW

**Comment by bitoiu** [comment]
Did you see this @jbjonesjr?

**Comment by jbjonesjr** [comment]
I did not.. but now I do.

File: README.md [hunk 1]
```diff
@@ -18,10 +18,45 @@ Download the files for installation
...
+ + Done!
+ + Creating fullchain.pem...
+ + Done!
+```
+
+
+### Certificate with alias
+
 ``` bash
+# The command below uses these flags
```
Comment (jbjonesjr) [inline README.md]: these additions are 🔥

File: README.md [hunk 2]
```diff
@@ -18,10 +18,45 @@ Download the files for installation
...
   $ git clone https://github.com/jbjonesjr/letsencrypt-manual-hook.git letsencrypt.sh/hooks/manual
+  $ cd letsencrypt.sh && chmod +x hooks/manual
 ```
 
 ## Usage
+
+### Single domain certificate
+
+``` bash
+$ ./letsencrypt.sh -c -t dns-01 -d looneytoons.acme.com -k ./hooks/manual/manual_hook.rb
```
Comment (jbjonesjr) [inline README.md]: we should add the same "explanation of switches used" in this example too

**Comment by bitoiu** [comment]
Will fix this soon :smile: thanks for the review!

**Comment by jbjonesjr** [comment]
@bitoiu hold off on this for a bit, due to some renaming of `letsencrypt.sh`, i'm updating my readme. merge conflicts are in the future for this PR, but i still love what's it's doing.

**Comment by jbjonesjr** [comment]
:wave: I merged #3, so let me know if you want to update this PR or if you want me to just move much of the work into the parent repo and save you the trouble.

**Comment by bitoiu** [comment]
> or if you want me to just move much of the work into the parent repo and save you the trouble.

If this is not too much work for you it might save us a bit of back and fw :bow:

**Pull request closed without merging**